core/templates/pages/landing-pages/stewards-landing-page/stewards-landing-page.component.html

```diff
@@ -378,7 +374,7 @@ <h2>
   }
 
   .oppia-stewards-dropdown {
-    margin-bottom: 50px;
+    margin-bottom: 1000px;
```

Comment:
Why this change?

Reply:
Oh, this was inadvertent, I'm sorry. I'll change it back!